app/dashboard/templates/quickstart.html

```diff
@@ -0,0 +1,169 @@
+{% comment %}
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n static %}
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    {% include 'shared/head.html' %}
+    {% include 'shared/cards.html' %}
+    <link rel="stylesheet" href={% static "v2/css/dashboard.css" %}>
+    <link rel="stylesheet" href={% static "v2/css/tag.css" %}>
+    <link rel="stylesheet" href={% static "v2/css/quickstart.css" %}>
+  </head>
+  <body class="interior {{active}} quickstart">
+    {% include 'shared/tag_manager_2.html' %}
+    <div class="container-fluid header dash">
+      {% include 'shared/nav.html' %}
+    </div>
+    <div class="container-fluid content mt-3">
+      <div class="row pb-1">
+        <div class="col text-center">
+          <h4 class="title">{% trans "Quickstart Guide" %}</h4>
+        </div>
+      </div>
+      <div class="row col-12 offset-lg-1 col-lg-10">
+        <div class="dont-show">
+          <div class="checkbox_container">
+            <input name="dontshow" id="dontshow" type="checkbox" />
+            <span class="checkbox"></span>
+            <div class="filter-label">
+              <label for=dontshow>{% trans "Don't show again" %}</label>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row col-12 offset-lg-1 col-lg-10 mt-2 steps-container">
+        <div class="col-md-3 col-sm-12 text-center p-2 image-container">
+          <img src={% static "v2/images/quickstart-box.svg" %}>
+        </div>
+        <div class="col-md-6 col-sm-12 p-5 steps-list">
+          <ol>
+            <li>{% trans "1) Create a Github Issue, you'll need the issue URL" %}</li>
+            <li>{% trans "2) Select the type of issue to fund" %}</li>
+            <li>{% trans '3) Complete the form and press "Fund Issue"' %}</li>
+            <li>{% trans "4) Confirm the transaction in metamask" %}</li>
+          </ol>
+          <p>
+            {% trans "Once confirmed on blockchain, your bounty is posted to Gitcoin!
```

Comment:
You'll want to be careful using `{% trans "" %}` spanning multiple lines with heavy whitespace between line segments.